lib/src/animated_play_pause.dart

```diff
@@ -37,6 +37,12 @@ class AnimatedPlayPauseState extends State<AnimatedPlayPause>
     }
   }
 
+  @override
+  void dispose() {
+    super.dispose();
```

Comment:
super.dispose should be called last.
Cannot find the best practices for it, but disposing super first could kill everything before we dispose our stuff (at least that's what I am thinking)

Reply:
I changed this already and will merge now.

Reply:
Thanks!